net/iodine/patches/100-musl-compatibility.patch

```diff
@@ -20,7 +20,7 @@ index 2715979..5f0e370 100644
  
  /* daemon(3) exists only in 4.4BSD or later, and in GNU libc */
 -#if !defined(ANDROID) && !defined(WINDOWS32) && !(defined(BSD) && (BSD >= 199306)) && !defined(__GLIBC__)
-+#ifdef __UCLIBC__
++#if 0
```

Comment:
Hmm, if there is a sane way to test the case "musl provides daemon" using cpp macros adapting the expression that is removed above accordingly would be better. (And then also adapt the comment.)

Reply:
This patch: https://github.com/CZ-NIC/turris-os/blob/test/patches/packages/iodine-fix.patch

uses __NO_DAEMON__. I can change it to that. I doubt the build system actually checks this variable but it is more verbose than a simple if 0.